components/src/core/score-card/scorecard.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+    selector: 'pxb-scorecard',
+    template: `
+        <mat-card class="root">
+            <div class="header" [style.backgroundColor]="headerColor" [style.color]="headerFontColor">
+                <div class="header-overlay"></div>
+                <div class="header-wrapper">
+                    <div class="header-text">
+                        <mat-card-title class="title">{{ headerTitle }}</mat-card-title>
+                        <mat-card-subtitle class="subtitle">{{ headerSubtitle }}</mat-card-subtitle>
+                        <mat-card-subtitle class="info">{{ headerInfo }}</mat-card-subtitle>
+                    </div>
+                    <div class="action-items">
+                        <ng-content select="[action-items]"></ng-content>
+                    </div>
+                </div>
+            </div>
+            <mat-card-content>
+                <div class="body">
+                    <ng-content select="[body]"></ng-content>
+                    <div class="badge-wrapper" [style.marginTop.px]="badgeOffset || 'inherit'">
```

Comment:
these wrappers still render even if no badge, etc. is supplied...how do we feel about that?

Reply:
Since it's an empty div it shouldn't cause any issues.  Do you have any concerns about this?

Reply:
https://github.com/angular/angular/issues/26083#issuecomment-425227938 . This is an alternative.

Reply:
My only concern is if the empty containers have styling associated with them (e.g., margins) that would affect the positioning of other elements.

Reply:
This particular instance doesn't have any styles on it so I think it should be fine.  If we have to make this a conditional render, we can go with the same approach used to conditionally render the mat-divider if there's no actionRow.